### PR TITLE
🚀  Release v1.6.6

### PR DESCRIPTION
**Pull request by vishalanarase**
/area release

### CODE REVIEW

**Comment by adilGhaffarDev** [comment]
/hold
LGTM from my side. Holding so others can also review.

**Review by adilGhaffarDev — APPROVED** [review]
/approve

File: CHANGELOG/v1.6.6.md
Comment (sbueringer) [inline CHANGELOG/v1.6.6.md:26]:
```suggestion
```
This PR was merged against main. If this was generated with the release notes tool we should take a look if we have a bug

(I didn't double check all other PRs if they were actually merged against release-1.6)

Comment (sbueringer) [reply CHANGELOG/v1.6.6.md:26]: /hold

Comment (sbueringer) [reply CHANGELOG/v1.6.6.md:26]: /hold cancel
Double checked all other PRs, the other ones are from release-1.6. But to be honest I don't have the time to do this :)

Comment (sbueringer) [reply CHANGELOG/v1.6.6.md:26]: Just ran the tool locally and indeed it adds the PR from main. Let's open an issue and investigate this please

Comment (adilGhaffarDev) [reply CHANGELOG/v1.6.6.md:26]: I will check the release notes again and also keep it in mind for next releases. I will create an issue and also investigate why this is happening.

Comment (sbueringer) [reply CHANGELOG/v1.6.6.md:26]: @vishalanarase Otherwise lgtm once this line is removed from my side :)

Comment (vishalanarase) [reply CHANGELOG/v1.6.6.md]: Updated the MR

**Comment by adilGhaffarDev** [comment]
@chandankumar4 @willie-yao please take a look. if it looks ok, I can unhold already.

**Review by chandankumar4 — APPROVED** [review]
/lgtm

**Pull request merged**
